src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -1056,9 +1069,17 @@ void mongoSubCacheStatisticsGet(int* refreshes, int* inserts, int* removes, int*
   {
     while (cSubP != NULL)
     {
+      char msg[256];
+
       unsigned int bytesLeft = listSize - strlen(list);
 
-      if (strlen(cSubP->subscriptionId) + 2 > bytesLeft)
+#if UpdateBugTests
```

Comment:
So the plan is to remove this after debugging, finding and fixing the bug? In that case, I'd suggest to add a FIXME comment about it just before the `#if`.